### PR TITLE
Fix local episode file not detected when status is on snatched

### PR DESCRIPTION
**Pull request by fernandog**
- [x] Add snatch tests

### CODE REVIEW

**Comment by fernandog** [comment]
can someone took a look a this please?
it does all that when refreshing files from disk

**Comment by fernandog** [comment]
@OmgImAlexis can you do some tests with this? old code was missing a lot of logs so user never knew whats going on

**Comment by OmgImAlexis** [comment]
So if a file is downloaded already but has a lower quality than the snatch it doesn't update, correct?

**Comment by fernandog** [comment]
@OmgImAlexis from the existing code in develop and in my PR, yes.

File: medusa/tv.py [hunk 1]
```diff
@@ -1075,75 +1108,17 @@ def make_ep_from_file(self, filepath):
...
+                with cur_ep.lock:
+                    old_ep_status = cur_ep.status
+                    cur_ep.status = Quality.composite_status(DOWNLOADED, new_quality)
+                    logger.log(u"{0}: Setting the status from '{2}' to '{3}' based on file: {4}. Reason: {5}".format
+                               (self.indexerid, statusStrings[old_ep_status], statusStrings[cur_ep.status],
+                                filepath, should_refresh_reason), logger.DEBUG)
             else:
-                logger.log(u"{0}: Old episode is '{1}'. Ignoring file: {2}".format
-                           (self.indexerid, statusStrings[cur_ep.status], filepath), logger.DEBUG)
+                logger.log(u'{0}: Not chag')
```
Comment (fernandog) [inline medusa/tv.py]: ops!

**Comment by fernandog** [comment]
@OmgImAlexis waiting your feedback on this

**Comment by fernandog** [comment]
Fixes https://github.com/pymedusa/Medusa/issues/2072

**Pull request merged**